README.md

```diff
@@ -97,138 +97,148 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+ - Fixed a crash when entering a duplicate name for a color in the Color Picker page and improved clean up when cancelling a color edit. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
-### Settings
+### Text Extractor
 
- - Fixed issue causing problems with modifier keys and ShortcutControl. Thanks [@sh0ckj0ckey](https://github.com/sh0ckj0ckey)!
- - Fixed crash when clicking "Windows color settings" link.
- - Added support for launching Settings app directly.
- - Fixed issue causing DisplayDescription not showing for PowerToys Run PluginAdditionalOption.
- - Fixed issue causing FileLocksmith 'Show File Locksmith in' setting not showing correct value.
- - Fixed issue causing Awake on/off toggle in Settings flyout not to work when Settings Awake page is opened.
+ - Text Extractor 2.0, with a new overlay, table mode and more Quality of Life improvements. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
 
 ### Documentation
 
- - Added documentation for PowerToys Run third-party plugins. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Fixed broken links in keyboardmanagerui.md. Thanks [@shubhsardana29](https://github.com/shubhsardana29)!
- - Updated core team in COMMUNITY.md.
- - Fixed broken links in ui-architecture.md. Thanks [@SamB](https://github.com/SamB)!
- - Updated community.valuegenerator.md with Base64DecodeRequest description.
+ - SECURITY.md was updated from 0.0.2 to 0.0.9. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
+ - Improved the README and main development document for clarity and completeness. Thanks [@codeofdusk](https://github.com/codeofdusk) and [@aprilbbrockhoeft](https://github.com/aprilbbrockhoeft)!
 
 ### Development
 
- - Updated test packages and StyleCop. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Condense NuGet Restore into MSBuild Tasks. Thanks [@snickler](https://github.com/snickler)!
+ - Fixed PowerToys Run DateTime plugin tests that were failing depending on locale, so that they can be run correctly on all dev machines.
+ - Fixed PowerToys Run System plugin tests that were failing for certain network interfaces, so that they can be run correctly on all dev machines. Thanks [@snickler](https://github.com/snickler)!
+ - Fixed a markdown bug on the GitHub /helped command.
+ - Switched build pipelines to a new agent pool. Thanks [@DHowett](https://github.com/DHowett)!
+ - New .cs files created in Visual Studio get the header added automatically. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
-#### What is being planned for version 0.74
+#### What is being planned for version 0.75
 
-For [v0.74][github-next-release-work], we'll work on the items below:
+For [v0.75][github-next-release-work], we'll work on the items below:
 
  - Language selection
- - Modernize and refresh the UX of PowerToys based on WPF
+ - .NET 8 upgrade
```

Comment:
```suggestion
 - .NET 8 upgrade
 - Policy support for managing PowerToys Run plugins.
*Attention*: A breaking change is planned (for 0.75), in which each plugin has to declare its identifier programmatically so that it can be controlled through GPO. For third-party plugin developers, please check https://github.com/microsoft/PowerToys/pull/27468 for more details.
```

Reply:
There happend a merge conflict.